client/me/purchases/non-primary-domain-dialog/index.jsx

```diff
@@ -77,6 +77,23 @@ class NonPrimaryDomainDialog extends Component {
 								},
 							}
 						) }
+						{ hasSetupAds &&
+							translate(
+								'{{br/}}{{br/}}You will also be ineligible for the WordAds program. Visit {{a}}our FAQ{{/a}} to learn more.{{br/}}{{br/}}',
```

Comment:
The BR line break tags probably don't need to be translated and it will make any future modifications to this string harder (if we ever want to increase or decrease the spacing, the whole string will have to go through a two week translation process and many hours of work for translators). Can you put them outside the string?